pom.xml

```diff
@@ -37,6 +37,17 @@
       <artifactId>spring-boot-starter-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
```

Comment:
You do not need it. It is a part of `spring-boot-starter-test`. It is redundant.